server/config.ts

```diff
@@ -37,7 +37,7 @@ export default {
   version: Date.now().toString(),
   production,
   testMode: process.env.NODE_ENV === 'test',
-  port: get('PORT', 3000),
+  port: Number(process.env.PORT) || 3000,
```

Comment:
what's the reason for this change? the `get` function pulls the port from the environment in the same way, but with a bunch more checks and error handling

Reply:
hmm, the compiled javascript for `get` (for me) is
```
function get(name, fallback, options = { requireInProduction: false }) {
    const fromEnv = process.env[name];
    if (fromEnv !== undefined) {
        return fromEnv;
    }
```
which doesn't seem to explicitly sanitise the input -- was trying to ensure the string coming from the environment is explicitly treated as a number and any non-numeric values are thrown away

would `Number(get('PORT', 3000))` make sense in this context?

note: `Number(process.env.REDIS_PORT) || 6379,` is also a candidate to a similar change

Reply:
done in a269aedd7e5391afed2a01707b7fcecb7d37558b